algolia/search/requests_indexing.go

```diff
@@ -42,3 +42,8 @@ type getObjectsReq struct {
 type batchReq struct {
 	Requests []BatchOperation `json:"requests,omitempty"`
 }
+
+type PartialUpdateOperation struct {
+	Operation string      `json:"_operation"`
```

Comment:
We could define a type to restrict the given operations? I don't know how it's done elsewhere in the Go client.

Reply:
Mix of both, but we tend not to have dedicated string-aliased types for those, since it would mean having to do a new commit/PR/merge/release every time there's a new value on the engine' side.